source/models/xlnet.py

```diff
@@ -0,0 +1,63 @@
+from pytorch_transformers import XLNetModel, XLNetConfig
```

Comment:
nice addition of XLNET for this project, it's a logical next step from a BERT model. Curious to hear how the results compare.